catalog/core/catalog-core-api/src/main/java/ddf/catalog/content/data/ContentItem.java

```diff
@@ -38,7 +38,7 @@
 
   String DEFAULT_FILE_NAME = "content_store_file.bin";
 
-  String QUALIFIER = "qualifier";
+  String QUALIFIER_KEYWORD = "qualifier";
```

Comment:
Missed this the first time through but this is part of the catalog api so we unfortunately can't change this constant name. DDF 3.0 maybe?

Reply:
I don't this this class is whitelisted: http://artifacts.codice.org/service/local/repositories/releases/content/ddf/docs/2.11.4/docs-2.11.4-Documentation.html#_catalog_whitelist.